VTEX - Budgets API.json

```diff
@@ -0,0 +1,5235 @@
+                            "type": "string",
+                            "description": "Context type."
+                          },
+                          "contextId": {
+                            "type": "string",
+                            "description": "Identification of the context type."
+                          }
+                        }
+                      }
+                    }
+                  }
+                },
+                "example": {
+                  "pagination": {
+                    "current": 1,
+                    "pages": 1,
+                    "results": 1
+                  },
+                  "transactions": [
+                    {
+                      "id": "b0e55425-a981-4d00-b6a2-9c7294d3d2b3",
+                      "budgetId": "019833db-27ad-7264-b104-dfed7c0fcbac",
+                      "allocationId": "324b5750-c827-4a6b-a670-c9297c832f4e",
+                      "type": "DEBIT",
+                      "amount": 528,
+                      "date": "2025-05-17T07:01:44.0196090Z",
+                      "contextType": "UNIT",
+                      "contextId": "093f29ac-7a1b-4d6b-a79a-48f89cf73d14"
+                    }
+                  ]
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/responseError"
+                },
+                "example": {
+                  "error": "Invalid request parameters."
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Unauthorized",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/responseError"
+                },
+                "example": {
+                  "error": "Authentication required."
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "Not Found",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/responseError"
+                },
+                "example": {
+                  "error": "Budget not found."
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/responseError"
+                },
+                "example": {
+                  "error": "An internal server error occurred."
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "VtexIdclientAutCookie": []
+          }
+        ]
```

Comment:
Remove it, the authorization is declared in the securitySchemes. Don't forget to delete the comma above as well.